.github/workflows/list_components.yml

```diff
@@ -4,7 +4,9 @@
 
 name: list_components
 
-on: workflow_dispatch
+on:
```

Comment:
The output of list_components should be published on github pages and linked from the README.

Reply:
I will add publishing to github pages in the next update.